app/src/org/commcare/fragments/SelectInstallModeFragment.java

```diff
@@ -128,6 +145,11 @@ public void onClick(View v) {
         return view;
     }
 
+    private void onErrorViewButtonClicked() {
```

Comment:
Can we make it a public static and put it somewhere where all 3 use-cases utilize it, may be in MessageActivity itself.

Reply:
Good call. Done